### ISSUE TITLE
why is the metronome mute?

### ISSUE THREAD

**Issue by FabioNevesRezende** [pterodactylus42/tuxguitar#11]
Why I can't use the metronome when my sheet got only one instrument tab like a guitar?
If I take this file for example (attached) it got only one track which is a guitar then the metronome is mute
I have to manually insert an useless track with drum in order to make the metronome work

![image](https://github.com/pterodactylus42/tuxguitar-2.0beta/assets/42902918/64a193fd-d484-40d3-a0a7-ab25617e3bf9)

**Comment by hapla** [comment]
Try adding a percussion instrument to the track. Right click on your instrument "Unnamed 1" -> Properties -> Settings of the instrument (pen symbol) -> Click + to add another instrument -> Click the drum symbol to make it a Percussion Channel -> Close the instruments setting window -> Close Properties window. This seems to work for me.

**Comment by FabioNevesRezende** [comment]
> Try adding a percussion instrument to the track. Right click on your instrument "Unnamed 1" -> Properties -> Settings of the instrument (pen symbol) -> Click + to add another instrument -> Click the drum symbol to make it a Percussion Channel -> Close the instruments setting window -> Close Properties window. This seems to work for me.

Yes I know that adding a percussion track makes the metronome audible but my question was really about why it is necessary? it should be nice to have a metronome working regardless of a percussion channel or not (like guitar pro)
